source/MapPanel.h

```diff
@@ -174,6 +179,9 @@ class MapPanel : public Panel {
 private:
 	// This is the coloring mode currently used in the cache.
 	int cachedCommodity = -10;
+	// Minimum and maximum color ranges used by MapColor for the custom outfitter prices.
+	double minColor = 1.;
+	double maxColor = 1.;
```

Comment:
Other keys have color ranges. Why does only the custom outfitter key need these?

Reply:
Because the other keys never have differences as big as 1 to 1 million, which is totally possible here, with all the gradiants between the two

Not doing this means we will have a legend not taking the discrepencies into account meaning it would have to assume a "usual price range"

Reply:
I removed all UI changes to greatly simplify the PR, as requested.

Reply:
Well, almost anyway, just gotta remove -0.5 from the value I recon and it'll be fine

I'd just like to point out that trades do have access to the min and max values to draw the UI properly:
`value = (2. * (price - com.low)) / (com.high - com.low) - 1.;`
line 813 of mappanel.cpp